### DISCUSSION TITLE
apple.com is not rendered correctly

### DISCUSSION THREAD

**Discussion by danielfalbo** [qutebrowser/qutebrowser#5807] [General] [answered]
![Screen-Recording-2020-10-16-at-1](https://user-images.githubusercontent.com/39460524/96263788-b4207f00-0fc3-11eb-865a-25c79f92ca81.gif)
This really does not create any problem to my experience, I rarely navigate apple's website, anyways someone may care about this

**Comment by The-Compiler** [comment]
What page is this on exactly? It doesn't seem to be https://www.apple.com/iphone-12/ - right? Also, can you show your `:version` info?

**Comment by danielfalbo** [comment, marked as answer]
Yes, it is apple.com/iphone-12 <http://apple.com/iphone-12> but probably there also are other pages like that. Here my version info:

qutebrowser v1.13.1
Git commit: a99f61071 on HEAD (2020-07-17 17:00:38 +0200)
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.0

**Reply by danielfalbo** [reply]
The website also seems to change based on the window size (or ratio, I don't know)

**Reply by rien333** [reply]
Actually, this page renders normally for me.

**Reply by danielfalbo** [reply]
Oh, thank you, so I’ve probably messed something up

**Reply by danielfalbo** [reply]
Closing this 👍

**Reply by rien333** [reply]
Well, we've a similar configuration, but I'm on Qt 5.15.1, if that matters.
